website/packages/ui/src/components/anchor.tsx

```diff
@@ -28,6 +28,7 @@ export const Anchor = ({ children }: { children: string | string[] }): JSX.Eleme
   const context = useContext(AnchorContext);
   const ref = useRef<HTMLElement | null>(null);
 
+  console.log(children);
```

Comment:
Is this `console.log` intended to be here?

Reply:
oh god 😆 let me remove that